lib/ngx/ssl.md

```diff
@@ -299,6 +300,43 @@ This function can be called in whatever contexts where downstream https is used.
 
 [Back to TOC](#table-of-contents)
 
+client_addr
+---------------
+**syntax:** *addr, err = ssl.client_addr()*
+
+**context:** *any*
+
+Returns the client ip address in the current SSL connection.
```

Comment:
What is the actual representation of the IP address? Is it in raw binary form or in textual representation? We should be explicit here. Also, the client address may not be an IP address. It could be a unix domain socket client which does not use the IP protocol at all.